recipes/pyranha/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "0.8" %}
+
+package:
+  name: pyranha
+  version: {{ version }}
+
+source:
+  fn: piranha-{{ version }}.tar.gz
+  url: https://github.com/bluescarni/piranha/archive/v{{ version }}.tar.gz
+  sha256: 649f91ceee56446f903551f0665b8807b833d91fd9622c412eadc95d353ac8e6
+
+build:
+  number: 0
+  skip: true        # [win and py3k]
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+    - gmp
+    - mpfr
+    - boost     1.62.*
+    - bzip2     1.0.*
+    - zlib      1.2.*
+    - piranha   {{ version }}
+    - python
+  run:
+    - gmp
+    - mpfr
+    - boost     1.62.*
+    - bzip2     1.0.*
+    - zlib      1.2.*
+    - piranha   {{ version }}
+    - python
+
+test:
+  requires:
+    - mpmath
+    - numpy
```

Comment:
@isuruf ok, seems reasonable. The way ``mpmath`` is used currently is that if you pass an ``mpf`` object to a pyranha function exposed from C++, it gets automatically translated into an MPFR float and passed into the C++ code. Similarly, C++ functions returning MPFR object are wrapped so that the MPFR object is turned into an ``mpf``.